SwedbankPaySDKTests/GoodWebViewRedirectsTests.swift

```diff
@@ -0,0 +1,93 @@
+import XCTest
+import Foundation
+@testable import SwedbankPaySDK
+
+class GoodWebViewRedirectsTests: XCTestCase {
+    private var redirects: GoodWebViewRedirects!
+    
+    private func parse(_ text: String) {
+        redirects = GoodWebViewRedirects(openDataFile: FileUtils.factory(text))
+    }
+    
+    private func expect(url: String, allowed: Bool) {
+        let expectation = self.expectation(description: "\(url) is \(allowed ? "" : "not ") allowed")
+        redirects.allows(url: URL(string: url)!) {
+            XCTAssertEqual($0, allowed, "Unexpected result for \(url)")
+            expectation.fulfill()
+        }
+    }
+    
+    private func waitForExpectations() {
+        waitForExpectations(timeout: 10, handler: nil)
+    }
+    
+    func testItParsesDomains() {
+        parse("example.com")
+        expect(url: "https://example.com/", allowed: true)
+        expect(url: "https://example.com/path", allowed: true)
+        expect(url: "https://sub.example.com/", allowed: false)
+        expect(url: "https://other.com/", allowed: false)
+        waitForExpectations()
+    }
+    
+    func testItParsesSingleWildcards() {
+        parse("*.example.com")
+        expect(url: "https://example.com/", allowed: true)
+        expect(url: "https://sub.example.com/", allowed: true)
+        expect(url: "https://sub.example.com/path", allowed: true)
+        expect(url: "https://double.sub.example.com/", allowed: false)
+        expect(url: "https://other.com/", allowed: false)
+        waitForExpectations()
+    }
+    
+    func testItParsesDoubleWildcards() {
+        parse("**.example.com")
+        expect(url: "https://example.com/", allowed: true)
+        expect(url: "https://sub.example.com/", allowed: true)
+        expect(url: "https://double.sub.example.com/", allowed: true)
+        expect(url: "https://double.sub.example.com/path", allowed: true)
+        expect(url: "https://other.com/", allowed: false)
+        waitForExpectations()
+    }
+    
+    func testItParsesComments() {
+        parse("#example.com")
+        expect(url: "https://example.com/", allowed: false)
+        expect(url: "https://other.com/", allowed: false)
+        waitForExpectations()
+    }
+    
+    func testEnsemble() {
+        parse("""
+# Comment
+domain.com
+ sub.domain.com
+
+#not.this.com
+*.subdomains.org  
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6137886)